Source/WebCore/page/Page.cpp

```diff
@@ -4986,4 +4986,12 @@ void Page::activeNowPlayingSessionUpdateTimerFired()
     chrome().client().hasActiveNowPlayingSessionChanged(hasActiveNowPlayingSession);
 }
 
+void Page::setLastAuthentication(LoginStatus::AuthenticationType authType)
+{
+    auto loginStatus = LoginStatus::create(RegistrableDomain(mainFrameURL()), "test"_s, WebCore::LoginStatus::CredentialTokenType::HTTPStateToken , authType, WebCore::LoginStatus::TimeToLiveAuthentication);
```

Comment:
I don't think we should land this with `"test"` always being the username we pass here. If it's easy to get the username from the caller of this function we should do that. Otherwise we should have a way to pass an empty/null string for now.